controllers/auth.js

```diff
@@ -0,0 +1,30 @@
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const User = require("../models/User");
+
+module.exports = {
+  login: async (request, response) => {
+    const {username, password} = request.body;
+
+    const user = await User.findOne({username});
+    const passwordCorrect =
+      user === null ? false : await bcrypt.compare(password, user.passwordHash);
```

Comment:
```suggestion
      user ? await bcrypt.compare(password, user.passwordHash) : false;
```
It's usually easier to think in positives. It requires extra brain power to figure out double negative logic.